### ISSUE TITLE
Simplifying lock states to just unlocked and locked

### ISSUE THREAD

**Issue by ClaudiaMia** [Project-Pandora-Game/pandora#874]
The goal is to rework the current lock system by removing the three properties states and just having the states "unlocked" and "locked" for lock slots. All locks are afterwards only "in effect" if locked.

This change allows to more convenient use items in the room inventory that come with pre-configured locks in the lock module slots.
It also makes templates of an item including the locks work ideally.

**Comment by SandrinePDR** [comment]
Just some questions:
* If an item comes with a pre-configured lock, is it possible to exchange that lock for something else
* Do I understand correctly that an item with a lock but in the state "unlocked" can be changed freely?

**Comment by Jomshir98** [comment]
>     * If an item comes with a pre-configured lock, is it possible to exchange that lock for something else
Yes - pre-configured lock just means having an item template with the lock saved too. This works in a way, that you can simply omit the lock while spawning the template (without modifying the template itself), allowing you to add a different one instead afterwards.
There would be no change to this behavior in this proposal.

>     * Do I understand correctly that an item with a lock but in the state "unlocked" can be changed freely?
Yes. To simplify the description of what we want to achieve:
We want states "unlocked lock" and "no lock at all" to be treated the same - the lock in the slot affecting anything only when it gets actually locked.
